core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceMethod.java

```diff
@@ -669,10 +667,9 @@ static boolean aggregationRequired(AggregationStrategy strategy, HttpRequest req
          * Whether the request is available to be aggregated.
          */
         static boolean aggregationAvailable(HttpRequest req) {
-            final String contentType = req.headers().get(HttpHeaderNames.CONTENT_TYPE);
+            final MediaType contentType = req.headers().contentType();
```

Comment:
Could we need to catch an exception from here not to be raised it to the caller?

Reply:
Ah I see it https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/common/MediaTypeJsonDeserializer.java#L54
or 
https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncodedResponse.java#L167

Just curious is why do we want to hide this exception?

Reply:
From the annotated http service point of view, I think the user's method should be executed regardless of the media type, especially for the cases which are not using content type negotiation. So ignoring invalid content type is simple and reasonable way to do it.